### ISSUE TITLE
Feature Request: Prevent useless import aliases

### ISSUE THREAD

**Issue by Diaoul** [astral-sh/ruff#13777]
Following a refactor renaming an alias, I ended up with this kind of import statement

```python
from myapplication.queuing import Queue as Queue
```

**Current behavior:** This passes ruff linting and formatting
**Expected behavior:** Autocorrected

Do you think this could be implemented in ruff?

**Comment by sbrugman** [comment]
Isn't this case already being covered by [useless-import-alias (PLC0414)](https://docs.astral.sh/ruff/rules/useless-import-alias/#useless-import-alias-plc0414) (opt-in via pylint)

**Comment by Diaoul** [comment]
Oops, can't believe I missed this... Sorry for the noise 🙈
